src/compiler/proposals-and-stabilization/custom-test-jobs.md

```diff
@@ -0,0 +1,96 @@
+# Adding custom test jobs or adding custom test components to rust-lang/rust CI
```

Comment:
nit: there's line-wrapping on the other files in our section, could you hard wrap this file too?

Reply:
Hard-wrapped to 100 columns in an intermediate commit before (round 2).